proto/v1beta1/grafeas.proto

```diff
@@ -293,9 +294,11 @@ message Note {
     grafeas.v1beta1.discovery.Discovery discovery = 15;
     // A note describing an attestation role.
     grafeas.v1beta1.attestation.Authority attestation_authority = 16;
+    // A note describing an in-toto link.
+    grafeas.v1beta1.intoto.InToto intoto = 17;
```

Comment:
We also want to define an intoto occurrence, as in https://github.com/grafeas/grafeas/blob/master/proto/v1beta1/grafeas.proto#L229